helpers/config.js

```diff
@@ -5,11 +5,16 @@ module.exports = {
 function getBootstrapConfig() {
   return {
     authRequired: strToBool(process.env.AUTH_REQUIRED, true),
-    cors: { headers: (process.env.CORS_HEADERS || '').split(',').filter(Boolean) },
+    cors: { headers: strToArray('CORS_HEADERS') },
     mockToken: strToBool(process.env.MOCK_TOKEN, false),
+    ignoreAuthRoutes: strToArray('IGNORE_AUTH_ROUTES'),
   };
 }
 
 function strToBool(text, defaultValue) {
   return text ? text.toLowerCase() === 'true' : defaultValue;
-}
\ No newline at end of file
+}
+
+function strToArray(envVarName) {
+  return (process.env[envVarName] || '').split(',').filter(Boolean);
+}
```

Comment:
This name is misleading. I would expect this function to return `['f', 'o', 'o']` when called with `'foo'` args.

Reply:
@mikolaj-leszczynski  You would call .toString() on array instead of using a dedicated function - having a dedicated function tells that it is doing something else and I won't expect seeing such function to return result of String#toString.
Hint: In review when you find some name to be not good enough, then always suggest some alternative to show that there is one.

@lucasrabiec Nitpick: `strToArray` -> `commaDelimitedStringToArray`

Reply:
@duszans 
>Hint: In review when you find some name to be not good enough, then always suggest some alternative to show that there is one.

:ok_hand: 

> You would call .toString() on array instead of using a dedicated function to get such result - having a dedicated function tells that it is doing something else and I wouldn't expect seeing such function to return result of String#toString.

But we are talking here about making an array out of a super specific string - not the other way. The method `strToArray` is not the override method for `toString.`